pkg/trace/agent/agent.go

```diff
@@ -491,3 +491,12 @@ func newEventProcessor(conf *config.AgentConfig) *event.Processor {
 func (a *Agent) SetGlobalTagsUnsafe(tags map[string]string) {
 	a.conf.GlobalTags = tags
 }
+
```

Comment:
you can discard this change, this is no longer require as this function is now inlined (ref : https://github.com/DataDog/datadog-agent/pull/10035/files#diff-c2dc8d00a060f89f4cd810fa2496eb721f7da1a6f538c4b2ec5c5f7a894a76e7R253)